src/applications/vaos/tests/e2e/va-request.cypress.spec.js

```diff
@@ -79,7 +79,7 @@ function fillOutForm(facilitySelection) {
       .to.have.property('optionDate3')
       .to.equal('No Date Selected');
 
-    expect(request.facility.facilityCode).to.eq('983GB');
+    expect(request.facility.facilityCode).to.contain('983');
```

Comment:
I don't think this is the right change to make, it is changing the test to test something other than what it was testing before.

Reply:
These tests fail locally for me with `eq('983GB')`. The `facilityCode` is either `983` or `983GB` depending on the test, which is why I changed it.

Reply:
This line has been in our Cypress tests for a while in production and has not caused a problem. I think that suggests that something in your overall changes in this PR is causing an issue that hasn't been fully investigated.

Reply:
These tests fail locally on `master`. I think the Cypress update in this PR exposed an issue with the test.

Reply:
This test may have other problems, but in this case, it looks to me like the changes you're making are covering up another issue introduced in this PR. This test broke because it's now loading the wrong facility selection page. That's controlled by feature flags and some checks against user data, so I would suspect that something in those mocks has gone wrong. This test passes for me locally on the master branch (though sometimes you get a strange issue where Cypress doesn't want to select radio buttons, which I've been fixing by adding `.focus()` commands before the click).

Changing the logic and assertions of a test, when the application logic itself hasn't changed, is a big red flag when writing and updating tests, in my experience. That's why this jumped out to me.

Reply:
Thanks for looking this. I'm not familiar with these tests so I couldn't tell if this was an issue with the tests themselves or something else. In this case it appears to be a bug with using `cy.intercept()` instead of `cy.route()` in a global hook. I think Cypress was caching the tests when I was switching branches, which is why it was failing for me on `master`. After restarting the test runner the tests started passing for me again. I think I'll be able to refactor this PR without touching any VAOS tests.

Reply:
Awesome, thanks. If you have any questions about the VAOS tests or need help figuring out why something's not working, I'm happy to help.